test/__snapshots__/preset-mini.test.ts.snap

```diff
@@ -206,8 +206,11 @@ div:hover .group-\\\\[div\\\\:hover\\\\]-\\\\[combinator\\\\:test-4\\\\]{combina
 .peer:is(:placeholder-shown)~.peer-is-placeholder-shown\\\\:text-3xl,
 .peer:not(:placeholder-shown)~.peer-not-placeholder-shown\\\\:text-3xl{font-size:1.875rem;line-height:2.25rem;}
 .peer:not(:placeholder-shown)~.peer-not-placeholder-shown\\\\:text-2xl{font-size:1.5rem;line-height:2rem;}
+.text-\\\\[length\\\\:2em\\\\],
 .text-2em,
 .text-size-\\\\[2em\\\\]{font-size:2em;}
+.text-\\\\[length\\\\:calc\\\\(1em-1px\\\\)\\\\]{font-size:calc(1em - 1px);}
+.text-\\\\[length\\\\:var\\\\(--size\\\\)\\\\]{font-size:var(--size);}
```

Comment:
Why is it called length instead of `size`?

Reply:
Just following tailwinds'
https://tailwindcss.com/docs/adding-custom-styles#resolving-ambiguities